ops/stripe_plans.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var async = require("async");
+var currency_data = require("../data/currencies");
+var currencies = Object.keys(currency_data);
```

Comment:
I notice you using the underscore in filenames again. I don't really have an issue with it, other than we don't follow that naming convention. If you like the underscore then we can keep it.